components/persistor/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@haventech/persistor",
     "description": "A subclass of supertype that serializes to and reconstitutes from MongoDB or SQL databases",
     "homepage": "https://github.com/haven-life/persistor",
-    "version": "12.0.0",
```

Comment:
Lets point the MR to latest master, which is already published. 
Also this would be a feature MR not a bug fix bump.